conf/app.routes

```diff
@@ -16,9 +16,12 @@ GET         /account/sign-out                            controllers.auth.AuthCo
 GET         /account/signed-out                          controllers.auth.SignedOutController.onPageLoad
 GET         /unauthorised                                controllers.auth.UnauthorisedController.onPageLoad
 
-GET         /there-is-a-problem                          controllers.JourneyRecoveryController.onPageLoad(continueUrl: Option[uk.gov.hmrc.play.bootstrap.binders.RedirectUrl] ?= None)
+# TODO verify if necessary
+GET         /there-is-a-problem-                         controllers.JourneyRecoveryController.onPageLoad(continueUrl: Option[uk.gov.hmrc.play.bootstrap.binders.RedirectUrl] ?= None)
```

Comment:
@iservport URL seems to be not correct, instead  /there-is-a-problem- can we have /journey-recovery until we get the actual content